assets/_js/init/selfserve.js

```diff
@@ -17,7 +17,6 @@ OLCS.ready(function() {
   OLCS.searchFilter();
 
   //load govuk frontend
-  window.GOVUKFrontend.initAll();
   module.exports = window.cookieManager;
   //window.cookieManager.init(window.cookieConfig);
   OLCS.GOVUKversion = "4.3.0";
```

Comment:
Not sure it actually matter but let's bump this version up too incase someone finds it useful.